### ISSUE TITLE
Linux path case issue

### ISSUE THREAD

**Issue by paulmelis** [StereoKit/StereoKit#589]
(originally reported as https://github.com/StereoKit/StereoKit.Templates/issues/1, but does not seem to be related to the template)

## Description

On Linux, `skshaderc` can't be found due to it being in dir `tools` and not `Tools`:

```
melis@juggle 16:20:/tmp/t$ dotnet new sk-net
The template "StereoKit: .NET Core project" was created successfully.

Processing post-creation actions...
Restoring /tmp/t/t.csproj:
  Determining projects to restore...
  Restored /tmp/t/t.csproj (in 269 ms).
Restore succeeded.

melis@juggle 16:21:/tmp/t$ dotnet run
/home/melis/.nuget/packages/stereokit/0.3.7-preview.9/build/SKShaders.targets(54,3): error MSB3073: The command ""/home/melis/.nuget/packages/stereokit/0.3.7-preview.9/Tools/skshaderc" -d -e -t xge -i "/home/melis/.nuget/packages/stereokit/0.3.7-preview.9/Tools/include" -o "obj/Debug/net7.0/Assets/" "Assets/floor.hlsl"" exited with code 127. [/tmp/t/t.csproj]

The build failed. Fix the build errors and run again.
melis@juggle 16:21:/tmp/t$ ls -l /home/melis/.nuget/packages/stereokit/0.3.7-preview.9/Tools/skshaderc
ls: cannot access '/home/melis/.nuget/packages/stereokit/0.3.7-preview.9/Tools/skshaderc': No such file or directory
melis@juggle 16:21:/tmp/t$ ls -l /home/melis/.nuget/packages/stereokit/0.3.7-preview.9/
total 46080
drwxr-xr-x 4 melis users     4096 Feb  7 11:36 android_openxr_loaders
drwxr-xr-x 2 melis users     4096 Feb  7 11:36 build
drwxr-xr-x 2 melis users     4096 Feb  7 11:36 images
drwxr-xr-x 3 melis users     4096 Feb  7 11:36 lib
drwxr-xr-x 9 melis users     4096 Feb  7 11:36 runtimes
-rw-r--r-- 1 melis users 47149853 Feb  7 11:36 stereokit.0.3.7-preview.9.nupkg
-rw-r--r-- 1 melis users       88 Feb  7 11:36 stereokit.0.3.7-preview.9.nupkg.sha512
-rwxr--r-- 1 melis users      980 Feb  4 20:17 stereokit.nuspec
drwxr-xr-x 3 melis users     4096 Feb  7 11:36 tools
melis@juggle 16:21:/tmp/t$ ls -l /home/melis/.nuget/packages/stereokit/0.3.7-preview.9/tools/skshaderc
-rwxr--r-- 1 melis users 6695648 Jan 16 02:06 /home/melis/.nuget/packages/stereokit/0.3.7-preview.9/tools/skshaderc
```

Same for `include` dir path used.

## Platform / Environment

Linux, 0.3.7-preview.9

**Comment by maluoi** [comment]
Thanks for bringing this up! I feel like I've flipped the case on this folder a surprising number of times now, for this very reason! I'll dig into this a little deeper and see if I can figure why I have to keep doing this!

**Comment by maluoi** [comment]
Apparently the case on this one has been flipped at least 3 times before, I'm glad I finally recognized it this time! :D I _think_ I've tracked it down, but it can be a tedious one to test. I'll be running a few more checks on this to verify it!

_For reference, the previous commits:_
4e4118bda2e186e7051f2e611de7ec334808e744
d6239c065f931608379f7f1821b1a9453c37bb15
da48f658f7d378f947ea75d211cdbb09f7bc7f48
